lms/templates/navigation-edx.html

```diff
@@ -89,55 +91,58 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
     </ol>
 
       % if should_display_shopping_cart_func(): # see shoppingcart.context_processor.user_has_cart_context_processor
-        <ol class="user">
-          <li class="primary">
-            <a class="shopping-cart" href="${reverse('shoppingcart.views.show_cart')}">
-              <i class="icon fa fa-shopping-cart"></i> ${_("Shopping Cart")}
-            </a>
-          </li>
-        </ol>
+    <ol class="user">
+      <li class="primary">
+        <a class="shopping-cart" href="${reverse('shoppingcart.views.show_cart')}">
+          <i class="icon fa fa-shopping-cart"></i> ${_("Shopping Cart")}
```

Comment:
this icon font should have an ```aria-hidden=true``` attribute [See Accessibility Styleguide](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Accessibility#using-icon-fonts)